index.html

```diff
@@ -64,5 +65,192 @@ <h2><a href="https://www.linkedin.com/in/ivan-martinez-von-halle/" class="connec
          </div>
       </section>      
    </section>
+  <!-- portada section (end) -->
+
+    <!-- works section (init) -->
+   <section id="works" style="justify-items: center;"> 
```

Comment:
Kindly note that it is a requirement that you match the design as much as you can. Currently the cards in this section have too much margins. Please fix this
<img width="1680" alt="Screenshot 2022-09-24 at 09 58 25" src="https://user-images.githubusercontent.com/48546992/192088152-c1ce6cfd-fca8-4ad1-b4c9-1e7a6d7f0fb1.png">

- [ ] Your design
<img width="532" alt="Screenshot 2022-09-24 at 11 20 08" src="https://user-images.githubusercontent.com/48546992/192088288-d8fbbecd-49db-498d-a0c0-2a0a1206cc67.png">

- [ ] Figma design
<img width="259" alt="Screenshot 2022-09-24 at 11 20 14" src="https://user-images.githubusercontent.com/48546992/192088300-0876d5cc-2a62-439a-99a1-5cc011ab2616.png">